#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import (ParameterGenerator, int_t)

PACKAGE = 'flir_boson_usb'

gen = ParameterGenerator()

gen.add('point_x', int_t, 0, 'point x', 0, 0, 319)
gen.add('point_y', int_t, 0, 'point x', 0, 0, 255)
gen.add('max_temp_limit', int_t, 0, 'max temperature limit for 8bit image', 50, -100, 300)
gen.add('min_temp_limit', int_t, 0, 'min temperature limit for 8bit image', 20, -100, 300)

exit(gen.generate(PACKAGE, PACKAGE, 'BosonCamera'))
